pkg/ddl/job_scheduler.go

```diff
@@ -582,7 +582,7 @@ func (s *jobScheduler) transitOneJobStepAndWaitSync(wk *worker, jobCtx *jobConte
 				jobCtx.logger.Warn("check MDL info failed", zap.Error(err))
 				return err
 			}
-		} else {
+		} else if job.LastSchemaVersion > 0 {
```

Comment:
Will non multi-schema change DDL return 0 here?

Reply:
Yes, some DDLs also return 0 for `runOneJobStep`